nft_ingester/src/program_transformers/token/mod.rs

```diff
@@ -64,20 +69,26 @@ pub async fn handle_token_program_account<'a, 'b, 'c>(
             );
             db.execute(query).await?;
             let txn = db.begin().await?;
-            let asset_update: Option<asset::Model> = asset::Entity::find_by_id(mint)
+            let asset_update: Option<asset::Model> = asset::Entity::find_by_id(mint.clone())
                 .filter(asset::Column::OwnerType.eq("single"))
                 .one(&txn)
                 .await?;
-            if let Some(asset) = asset_update {
+            if let Some(_asset) = asset_update {
```

Comment:
So the other time that we use `upsert_assets_token_account_columns` we don't actually search the asset table. We search the token_accounts table. So even though they are both SQL searches, there isn't actually that much redundant logic here.

Reply:
And I think it would get kind of complicated to reuse the same upsert method because you don't actually want to insert here, and you wouldn't just add the condition to the on conflict part.

so probably if we wanted to combine the reads and upserts in this file to single SQL statements, they would be new update statements with the condition integrated.